cloudsploit/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.6-slim-buster
+
+LABEL maintainer="Axel Pavageau <axel.pavageau@ekino.com>"
+
+ADD package.json .
+
+RUN apt-get update -qq \
+    && pip install -U pip \
+    && pip install pipenv awscli boto3 \
+    && apt install -y npm git wget zip
+
+RUN npm install \
```

Comment:
```
npm install -g cloudsploit@^2.0
```
does not work?

Reply:
I think @axelpavageau did that to eventually let dependabot scan package.json and upgrade the dependency automatically.

If so, how about setting it up now @axelpavageau ?

Reply:
@Crow-EH is right, this is setup for dependabot to handle (although I haven't tested whether it's able to handle this specific dependency git/tag format).

I've added the missing dependabot configurations